example/index.js

```diff
@@ -81,13 +87,12 @@ scatterplot.subscribe('pointout', pointoutHandler);
 scatterplot.subscribe('select', selectHandler);
```

Comment:
Question not related to this PR: Not this line but the line above, does the casing of the the event name matter?

I'm asking because in #60 I used "pointOver" and "pointOut", and see "pointover" and "pointout" above.

Reply:
Good question! It does not matter because I made the pubSub instance case-insensitive. (See https://github.com/flekschas/regl-scatterplot/blob/master/src/index.js#L191)